src/app/features/test-records/components/vehicle-header/vehicle-header.component.ts

```diff
@@ -39,4 +39,24 @@ export class VehicleHeaderComponent {
   get vehicleTypes() {
     return VehicleTypes;
   }
+
+  createHeader(techRecord: TechRecordModel, vehicleType: VehicleTypes) {
```

Comment:
We're not really creating a header here, we're getting a vehicle description - perhaps this function name better describes what it does?

```suggestion
  getVehicleDescription(techRecord: TechRecordModel, vehicleType: VehicleTypes) {
```